AutomationHelpers/src/UserCodeCollections/WebLibrary.cs

```diff
@@ -66,5 +66,33 @@ public static class WebLibrary
                 Report.Log(ReportLevel.Warn, message);
             }
         }
+
+        /// <summary>
+        /// Return the http status code from an URL.
+        /// </summary>
+        /// <param name="url">The url from which the response code should be returned.</param>
+        [UserCodeMethod]
+        public static string GetHttpStatusCode(string url)
+        {
+            HttpWebResponse resp = null;
+
+            try
+            {
+                HttpWebRequest req = (HttpWebRequest)WebRequest.Create(url);
+
+                resp = (HttpWebResponse)req.GetResponse();
+            }
+            catch (WebException we)
+            {
+                if ((we.Response as HttpWebResponse).StatusCode == HttpStatusCode.NotFound)
+                {
+                    return "404";
```

Comment:
This is a bit weird since this is the only value returning a number now, isn't it?
All others return the enum value, i.e. a string like "OK" or "Redirect"

Reply:
fixed